content/blog/copilot-lessons/index.md

```diff
@@ -90,7 +90,7 @@ But while minimizing the LLM workload helped with efficiency, we soon faced an e
 
 Large language models excel at generating well-structured, grammatically correct output. They makes neat tables, tell good stories, and generally sound confident. That's what makes them dangerous because this polished presentation can mask underlying flaws in the information itself, creating a false sense of confidence for users.
 
-One of our early testers, Pablo, encountered this firsthand. He posed a query to Pulumi Copilot, asking for a summary of resources within a specific project. The response he received was impeccably formatted, neatly categorizing resources by type and providing counts for each. It *looked* right, and for us humans sometimes looking right carries a lot of weight.
+One of our early testers, Pablo, <!-- TODO: ask Pablo --> encountered this firsthand. He posed a query to Pulumi Copilot, asking for a summary of resources within a specific project. The response he received was impeccably formatted, neatly categorizing resources by type and providing counts for each. It *looked* right, and for us humans sometimes looking right carries a lot of weight.
```

Comment:
Must ask Pablo if he is OK to be mentioned, or leave the name out. cc: @adamgordonbell

Reply:
@sicarul Thoughts on being in here?

Reply:
Hey team! I'm fine with this, thanks for checking!